trust-center-agent/src/main/java/care/smith/fts/tca/consent/GicsPropertyNotEmptyCondition.java

```diff
@@ -0,0 +1,29 @@
+package care.smith.fts.tca.consent;
+
+import java.util.Arrays;
+import java.util.stream.StreamSupport;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Condition;
+import org.springframework.context.annotation.ConditionContext;
+import org.springframework.core.env.ConfigurableEnvironment;
+import org.springframework.core.env.EnumerablePropertySource;
+import org.springframework.core.type.AnnotatedTypeMetadata;
+
+@Slf4j
```

Comment:
Add a JavaDoc describing what the condition does.
Something like "spring context condition that checks..."